plugins/httpapi/zabbix.py

```diff
@@ -171,11 +171,9 @@ def send_request(self, data=None, request_method="POST", path="/api_jsonrpc.php"
             except ValueError:
                 raise ConnectionError("Invalid JSON response: %s" % value)
 
-            try:
-                # Some methods return bool not a dict in "result"
-                iter(json_data)
-            except TypeError:
-                # Do not try to find "error" if it is not a dict
+            # Some methods return bool or string(xml, yaml...)
```

Comment:
Please provide a file you are importing that would result in having `json_data` as `string` type. I tried all formats and it's always JSON.

Reply:
please import same xml template 2 times.
first importing will be success and second one will be failed.

Reply:
providing file will take time. because I use this plugin for internal use and the template is private.

Reply:
could you use this file?
https://github.com/ansible-collections/community.zabbix/issues/888#issuecomment-1637915841

Reply:
Indeed the first time it succeeds and the second time it fails. Investigating.

Reply:
I miss-understood! I read your PR.

This "error" checking have to be in previous of  "result" attribute extracting.
Nice PR!